packages/eas-cli/src/build/utils/formatBuild.ts

```diff
@@ -97,7 +98,28 @@ export function formatGraphQLBuild(build: BuildFragment): string {
             return null;
           case GraphQLBuildStatus.Finished: {
             const url = build.artifacts?.buildUrl;
-            return url ? url : chalk.red('not found');
+            return url ? link(url) : chalk.red('not found');
+          }
+          default:
+            return null;
+        }
+      },
+    },
+    {
+      label: 'Fingerprint',
+      get value() {
+        switch (build.status) {
+          case GraphQLBuildStatus.New:
+          case GraphQLBuildStatus.InQueue:
+          case GraphQLBuildStatus.InProgress:
+            return '<in progress>';
+          case GraphQLBuildStatus.PendingCancel:
+          case GraphQLBuildStatus.Canceled:
+          case GraphQLBuildStatus.Errored:
+            return null;
+          case GraphQLBuildStatus.Finished: {
+            const hash = build.fingerprint?.hash;
+            return hash ? chalk.green(hash) : chalk.red('not found');
```

Comment:
@sjchmiela I thought it would be good to highlight the fingerprint in green (same as build status) if it exists, and also added link highlighting to the URLs.

![Screenshot 2025-08-06 at 6.55.08 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/oymxNZblSIUJ37GzgGHk/885fbae9-cd03-44bc-be6b-5a852a7ed8fe.png)